frontend/app/utils/locale-utils.ts

```diff
@@ -7,12 +7,13 @@ import { type UNSAFE_RouteModules as RouteModules } from '@remix-run/react';
  */
 export function getLocale(url: string) {
   const pathname = new URL(url).pathname;
+  const locale = pathname.split('/')?.[1];
 
-  if (pathname.startsWith('/en')) {
+  if (locale === 'en') {
```

Comment:
Is locale case always be in lowercase? If someone type `/EN/home` is should works. Please check if `pathname` is always in lowercase.

Reply:
`/EN` returns a 404, as expected

Reply:
Its depends on the platform. Remix is case sensitive and Next.js is not based on the document sites.